openwisp_monitoring/check/migrations/0003_create_ping.py

```diff
@@ -23,6 +24,7 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('check', '0002_check_unique_together'),
+        swapper.dependency('monitoring', 'Metric'),
```

Comment:
This is required because the code uses Metric model. It serves as sanity check.

Reply:
Oh Okay! I will learn more about it.